dali/test/python/test_operator_transpose.py

```diff
@@ -37,7 +37,8 @@ def __init__(self, device, batch_size, layout, iterator, num_threads=1, device_i
         self.iterator = iterator
         self.inputs = ops.ExternalSource()
         self.transpose = ops.Transpose(device = self.device,
-                                       perm = (1, 0, 2))
+                                       perm = (1, 0, 2),
```

Comment:
Please add cases for:
- `transpose_layout =True`
- `output_layout = some_value`